wcivf/apps/parishes/models.py

```diff
@@ -19,7 +19,7 @@ class ParishCouncilElection(models.Model):
     )
     sopn = models.URLField(blank=True, help_text="Link to SoPN PDF")
     ward_seats = models.PositiveIntegerField(default=0)
-    is_contested = models.NullBooleanField(default=None)
+    is_contested = models.NullBooleanField(default=True)
```

Comment:
The change needed here is to change to `BooleanField(null=True`.

This will also need a `makemigrations` run to generate the migration file and then be committed. Running this will also generate files for the change in `JsonField` elsewhere